frontend/src/components/Topbar.jsx

```
@@ -103,25 +82,28 @@ export const Topbar = () => {
+      </button>
+
       <div className={styles.rightSide}>
-        {/* Wrap notification in a container */}
+        {/* Notifications */}
         <div className={styles.notificationContainer}>
-          <div className={styles.notification} onClick={handleNotificationBtn}>
+          <button
+            className={styles.notification}
+            onClick={handleNotificationBtn}
+            aria-label="Notifications"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)